docs/editor/workspaces.md

```diff
@@ -53,6 +53,8 @@ You don't have to do anything for a folder to become a VS Code workspace other t
 
 ![Multi-root workspace](images/workspaces/multi-root-workspace.png)
 
+Multi-root workspaces are very useful to implement the loose definition of "project" stated above.
```

Comment:
I don't think we need to introduce a definition of a "project" so we can remove this sentence.

Reply:
👍